portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase

```diff
@@ -598,7 +598,6 @@ definition {
 	@description = "This is a use case for LPS-73381. LDAP security credentials are obscured when entered."
 	@priority = 3
 	test LDAPSecurityCredentialsAreHidden {
-		property portal.upstream = "quarantine";
```

Comment:
Looks like you remove from quarantine here, only to re-add it in the next commit.  And I don't see the test run in the security suite.  Could you take a look at this property and make sure you have everything correct?

Reply:
I messed up while committing this, my bad :sweat_smile: . I pushed again and ran the security suite to check the test